src/styles/components/header.scss

```diff
@@ -1,2 +1,78 @@
 .header {
+    background: var(--color-background-darker);
+    font-size: 2rem;
+    font-weight: 600;
+    border-bottom: 1px solid #5d5c61;
+    
+
+    &__content{
+        // height: 50px;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+    }
+
+    &__content-item{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-shrink: 5;
+        margin-left: 40px;
+        padding: 15px 0;
+    }
+
+    &__content-item:nth-of-type(2){
+        margin-right: 40px;
+        margin-left: 0px;
+    }
+
+    &__menu-item{
+        display: inline-block;
+        margin-left: 45px;
+        cursor: pointer;
+        text-decoration: none;
+        transform: translateY(1.3px);
+
+        &:visited{
+            color: white;
+        }
+
+        &:first-child{
+            margin-left: 0;
+        }
+
+        &:last-child{
+            color: var(--color-text-purple);
+        }
+    }
+
+
+    &__logo-link{
+        cursor: pointer;  
+        display: flex;
+        justify-content: center;
+        padding: 5px 0;
+    }
+
+    &__logo{
+        // width: 175px;
+        height: 20px;
+        pointer-events: none;
+
+    }
 }
+
+@keyframes menuItemSizeDecrease{
+    from {font-size: 2rem;}
+    to {font-size: 1.6rem}
+}
+
+@media (max-width: 600px) {
```

Comment:
Ill add a [Sass mixin](https://sass-lang.com/documentation/at-rules/mixin/) for breakpoints, then lets have the layout be mobile first and have variables for each breakpoint for example `--breakpoint-tablet` and `--breakpoint-desktop`

Reply:
Ill wait for your update on this one.

Reply:
I have pushed the mixin update with examples, this media query should become the default style for the component ( as we are using the responsive web design concept of mobile first)

Reply:
this means that all styles that apple mobile and up go under the default selector, styles that apply tablet and up go in the tablet breakpoint and styles that apply desktop and up go in the desktop breakpoint

Reply:
I have noted the implementation and usage.